sdk/python/tests/components/test_graph_components.py

```diff
@@ -172,5 +172,33 @@ def test_handle_parsing_task_volumes_and_mounts(self):
 
 #TODO: Test task name conversion to Argo-compatible names
 
+
+    def test_handle_parsing_nested_graph_component(self):
+        component_text = '''\
+name: Top-level component
+implementation:
+  graph:
```

Comment:
Could you remove the graph component in the test here, which is not in the scope of component yaml any more.

Reply:
I'm not sure how to test it in a better way.
I could probably test the PipelineRunSpec instead. Would that work for you?